plugin/pki/vcert.go

```diff
@@ -7,23 +7,22 @@ import (
 	"github.com/Venafi/vcert/v4/pkg/endpoint"
 	"github.com/hashicorp/vault/sdk/logical"
 	"io/ioutil"
-	"time"
 )
 
 func (b *backend) ClientVenafi(ctx context.Context, req *logical.Request, role *roleEntry) (
-	endpoint.Connector, *vcert.Config, time.Duration, error) {
+	endpoint.Connector, *vcert.Config, error) {
 
 	cfg, err := b.getConfig(ctx, req, role, false)
 	if err != nil {
-		return nil, nil, 0, err
+		return nil, nil, err
 	}
 
 	client, err := vcert.NewClient(cfg)
 	if err != nil {
-		return nil, nil, 0, fmt.Errorf("failed to get Venafi issuer client: %s", err)
+		return nil, nil, fmt.Errorf("failed to get Venafi issuer client: %s", err)
 	}
 
-	return client, cfg, role.ServerTimeout, nil
```

Comment:
Not sure why it was being returned in the first place. It can be easily obtained during the workflow using the `role.ServerTimeout` parameter, a change I'm doing above:
![image](https://user-images.githubusercontent.com/77068025/205767647-ad3d17b6-ba5e-4454-b946-0c37d4a454d5.png)